modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -62,7 +362,7 @@ def _read(cls, path, engine, columns, **kwargs):
             # parquet directories have a unique column at each directory level.
             # Thus, we can use os.walk(), which does a dfs search, to walk
             # through the different columns that the data is partitioned on
-            for (root, dir_names, files) in os.walk(path):
+            for (_, dir_names, files) in os.walk(path):
```

Comment:
Was this change made because `root` is no longer used?

Reply:
Yes